apps/docs/src/lib/config.ts

```diff
@@ -4,16 +4,26 @@ function isProduction() {
   return process.env.NODE_ENV === 'production';
 }
 
+function isUat() {
+  return process.env.NODE_ENV === ('uat' as any);
+}
+
+function isStaging() {
+  return process.env.NODE_ENV === ('staging' as any);
+}
+
 export type DocumentSiteConfiguration = {
   isProduction: () => boolean;
```

Comment:
These helpers could be removed, as they are not currently used, `isProduction`, `isUat`, `isStaging`